EventsExpress/ClientApp/src/actions/eventSchedule/eventSchedule-item-view-action.js

```diff
@@ -9,13 +10,14 @@ const api_serv = new EventScheduleService();
 
 export default function getEventSchedule(id) {
     return async dispatch => {
-        dispatch(getEventSchedulePending(true));
+        dispatch(getRequestInc());
         let response = await api_serv.getEventSchedule(id);
         if (!response.ok) {
             dispatch(setErrorAllertFromResponse(response));
             return Promise.reject();
         }
         let jsonRes = await response.json();
+        dispatch(getRequestDec());
```

Comment:
This line should go before the `if` otherwise there might be cases when spinner will not disappear.